scripts/get-version.js

```diff
@@ -5,71 +5,61 @@ const fetch = require('cross-fetch');
 
 const COMMIT_URL = 'https://api.github.com/repos/the-hideout/tarkov-dev/commits/main';
 
-// Use a GitHub token to increase the rate limit
+// If a GitHub token is provided, use it to increase the rate limit
 const token = process.env.GITHUB_TOKEN;
 const headers = {};
 if (token) {
     headers.authorization = `token ${token}`;
-    console.log("Using provided GitHub token to increase rate limit")
+    console.log("Using provided GitHub token to increase rate limit");
 } else {
-    console.log("No GitHub token provided, rate limit may be reached")
+    console.log("No GitHub token provided, rate limit may be reached");
+    console.warn("To increase the rate limit, provide a GitHub token via the GITHUB_TOKEN environment variable");
 }
 
-const getVersion = async function getVersion() {
+async function getVersion() {
     console.time(`Get version url ${COMMIT_URL}`);
+    let responseJson = {};
+
     try {
         const response = await fetch(COMMIT_URL, {
             headers,
             signal: AbortSignal.timeout(5000),
-        }).then(response => response.json());
-        console.timeEnd(`Get version url ${COMMIT_URL}`);
-
-        return response;
-    } catch (responseError) {
-        console.timeEnd(`Get version url ${COMMIT_URL}`);
+        });
 
-        if (responseError instanceof got.HTTPError) {
```

Comment:
@Razzmatazzz The reason I added back `got` was because I saw this line and noticed it would through errors because of `instanceof got.HTTPError`. If the `got` lib isn't available here, it looks like this line would blow up.

But... the amazing @Shebuka just swung on by and fixed all of that for us 🎉